source/NVDAObjects/IAccessible/MSHTML.py

```diff
@@ -975,6 +995,76 @@ def _get_language(self):
 			return ti.getControlFieldForNVDAObject(self)["language"]
 		except LookupError:
 			return None
+		
+	def getCustomLabel(self,nameAttribute):
+  		filename=self.getFilenameFromElementDomain()
+		config = configobj.ConfigObj(os.path.join(globalVars.appArgs.configPath, "webLabels\%s" % filename))
+		try:
+			for k,v in config.iteritems():
+				if (k==nameAttribute):
+					return v
+		except Exception as e:
+   			pass
+	
+	def getFilenameFromElementDomain(self):

```

Comment:
Perhaps move this function into its own custom web labels module and then it can be used from the base VirtualBuffer as well... I think there is some code duplication.